### PR TITLE
Update master from 1.2

### PR DESCRIPTION
**Pull request by dbu**
lets see if this makes all sense or if some of the new things somehow conflict.

### CODE REVIEW

File: Tests/Unit/Provider/PhpcrMenuProviderTest.php [hunk 1]
```diff
@@ -90,7 +94,45 @@ public function testHas($menuRoot, $name, $expectedPath)
             $menuRoot
         );
 
-        $provider->has($name);
+        $this->assertTrue($provider->has($name));
+    }
+
+    public function testHasNot()
```
Comment (lsmith77) [inline Tests/Unit/Provider/PhpcrMenuProviderTest.php]: likely needs to be updated for KnpMenu 2.0

**Comment by WouterJ** [comment]
Rebased and improved travis setup. Ready to be merged

**Comment by dbu** [comment]
did rebasing make sense? the idea is to make it possible to merge 1.2 into master. rebasing means that all those commits from 1.2 got a new commit id, so next time we try to merge 1.2 into master we risk to run into a big mess.

should i create a new branch and then port the fixes @WouterJ did here into that branch?

**Comment by WouterJ** [comment]
Ah, I'm not sure if Git can handle it. I've done a new merge in a new branch as requested: https://github.com/symfony-cmf/MenuBundle/pull/239

Closing in favor of that one.

**Pull request closed without merging**